jsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+    "compilerOptions": {
+        "target": "ES6",
+        "module": "commonjs"
+    }
+}
```

Comment:
This is for the VS Code editor instructing it to use ES6 syntax.

On Oct 27, 2015, at 9:51 AM, C.J. Winslow <notifications@github.com<mailto:notifications@github.com>> wrote:


In jsconfig.json<https://github.com/docusign/DocuSign-Node-Client/pull/36#discussion_r43150218>:

> @@ -0,0 +1,6 @@
> +{
> +    "compilerOptions": {
> +        "target": "ES6",
> +        "module": "commonjs"
> +    }
> +}


What's the purpose of this file?

-
Reply to this email directly or view it on GitHub<https://github.com/docusign/DocuSign-Node-Client/pull/36/files#r43150218>.